superset/cli.py

```diff
@@ -16,7 +16,7 @@
 from pathlib2 import Path
 import yaml
 
-from superset import app, db, dict_import_export_util, security, utils
+from superset import app, data, db, dict_import_export_util, sm, utils
```

Comment:
This could be out-of-scope for this PR, but it would be nice to always be explicit about `sm` and always call it `security_manager` (my bad!).

It may be reasonable to take it on as part of this PR.

Reply:
I don’t mind the shortened sm (inline with db and other variables) as it’s used frequently and is considerably shorter than security_manager (2 characters vs. 16).